go/vt/vttablet/tabletserver/state_manager.go

```diff
@@ -486,6 +495,7 @@ func (sm *stateManager) closeAll() {
 	sm.vstreamer.Close()
 	sm.rt.Close()
 	sm.se.Close()
+	sm.throttler.Close()
```

Comment:
This is already being called in unserveCommon.

Reply:
fixed